_data/simple-icons.json

```
@@ -9583,6 +9583,11 @@
             "source": "https://www.maptiler.com/press/#logo",
             "guidelines": "https://www.maptiler.com/press/#logo"
         },
+        {
+            "title": "Mapy.cz",
+            "hex": "1EAE00",
+            "source": "https://o-seznam.cz/ke-stazeni/loga"
```

The same page has brand guidelines.
```suggestion
            "source": "https://o-seznam.cz/ke-stazeni/loga",
            "guidelines": "https://o-seznam.cz/ke-stazeni/loga/"
```